byway/src/main/webapp/destinations.js

```diff
@@ -133,17 +129,50 @@ function addMarker(searchBox,map){
 * fetches start location and destinations from DestinationsServlet and adds to DOM
 */
 function updateLocations(locationData){
-    document.getElementById('start-location').innerText = "Start Location :" + locationData.start;
     const container = document.getElementById('destinations-container');
-    container.innerText = "Destinations:";
+    container.innerText = "";
     let destinationArray= locationData.destinations;
     destinationArray.forEach((destination) => {
-      let destinationToAdd = document.createElement('p');
-      destinationToAdd.innerText = destination;
-      container.appendChild(destinationToAdd);
+      const request = {
+        query: destination,
+        fields: ["name", "photos", "formatted_address", "rating", "business_status"]
+      };
+      let service = new google.maps.places.PlacesService(map);
+      service.findPlaceFromQuery(request, (results, status) => {
+        if (status === google.maps.places.PlacesServiceStatus.OK) {
+          addLocationtoDOM(results, container);
+        }
+        else{
+          alert("Location Invalid");
+        }
+      });  
     }) 
 }
 
+/** 
+* adds Users Input Destinations to DOM with image and address
+*/
+function addLocationtoDOM(results,container){
+  let destinationToAdd = document.createElement('div');
+  destinationToAdd.className = 'location';
+  
+  let destinationPhoto = document.createElement('img');
+  destinationPhoto.src = results[0].photos[0].getUrl();
+  let destinationInfo = document.createElement('div');
```

Comment:
For pure text, it makes sense to create `<p>` tags. My earlier comment was mainly for the containing section as a whole.

Reply:
ah gotcha! changed it back